stable/aws-sigv4-proxy-admission-controller/templates/webhook.yaml

```diff
@@ -0,0 +1,34 @@
+{{ $tls := fromYaml ( include "sidecar-injector.gen-certs" . ) }}
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: {{ template "sidecar-injector.fullname" . }}-webhook-cfg
+  labels:
+    app: {{ template "sidecar-injector.fullname" . }}
+{{ include "sidecar-injector.labels" . | indent 4 }}
+webhooks:
+  - name: {{ template "sidecar-injector.fullname" . }}.k8s.aws
+    clientConfig:
```

Comment:
You also need to add `sideEffects: None` so that it is compatible with newer kubernetes versions. v1beta1 is deprecated and will not be supported eventually.